deploy/role.yaml

```diff
@@ -40,6 +40,18 @@ rules:
   - statefulsets/finalizers
   verbs:
   - '*'
+- apiGroups:
+  - "route.openshift.io"
+  resources:
+  - routes
+  verbs:
+  - '*'
+- apiGroups:
+  - "extensions"
```

Comment:
@rigazilla is "extension" correct apiGroup. It's probably must be networking.k8s.io?
https://kubernetes.io/docs/concepts/services-networking/ingress/

Reply:
... and right :) 
I've used an on group for Ingress. I'll change also in the code.
Thanks again @dmvolod !